maintainers/README.md

```diff
@@ -0,0 +1,180 @@
+  active participation in project forums such as github, slack, or the
+  metal3-dev mailing list.
+
+- Plans for sustained contribution to the project compatible with the project's
+  direction as viewed by current maintainers
+
+The process to grant approval access to a candidate is as follows:
+
+- An existing maintainer nominates the candidate by sending an email to all
+  existing maintainers with information substantiating the contributions of the
+  candidate in the areas described above.
+
+- All existing maintainers discuss the pros and cons of granting approval
+  access to the candidate in the email thread.
+
+- When the discussion has converged or a reasonable time has elapsed without
+  discussion developing (e.g. a few business days) the nominator calls for a
+  final decision on the candidate with a followup email to the thread.
+
+- Each maintainer may vote yes, no, or abstain by replying to the email thread.
+  A failure to reply is an implicit abstention.
+
+- After votes from all existing maintainers have been collected or a reasonable
+  time has elapsed for them to be provided (e.g. a couple of business days) the
+  votes are evaluated. To be granted approval access the candidate must receive
+  yes votes from a majority of the existing maintainers and zero no votes. Since
+  a no vote is effectively a veto of the candidate it should be accompanied by
+  a reason for the vote.
+
+- The nominator summarizes the result of the vote in an email to all existing
+  maintainers
+
+- If the vote to grant approval access passed, the candidate is contacted with an
+  invitation to become a maintainer to the project which asks them to agree to
+  the maintainer expectations documented here.
+
+- If the candidate agrees access is granted by adding the new maintainer to the
+  appropriate `OWNERS` file(s).
+
+## Removing Approval Access Due to Inactivity
+
+The process for adding maintainers discusses a set of criteria which includes
+contributing to the project through code, reviews, and discussions.  It is
+normal that plans and assignments change over time and a maintainer may no
+longer have the capacity to continue contributing.  In that case, it is desired
+that the maintainer remove themselves from the appropriate `OWNERS` file(s).
+If participation picks up in the future, it should be easy to get re-added.
+
+Existing maintainers may also reach out to other maintainers to check on their
+availability for ongoing participation, prompting a discussion about whether it
+would be appropriate to drop out of the maintainers list.  If no response is
+received, an existing maintainer may propose removing a maintainer due to
+inactivity.  Again, it should be easy to re-add if participation picks up again
+in the future.
+
+Removal due to inactivity only needs to be approved by the maintainer being
+removed.  Otherwise, if the maintainer is unreachable, a maintainer may be
+removed with approval by two thrids of the existing maintainers.
```

Comment:
typo: thrids

Reply:
Also this seems like an unnecessarily high bar when the maintainer is inactive and unreachable.

Reply:
Yeah it seems like this is already covered pretty well via "If no response is
received, an existing maintainer may propose removing a maintainer due to
inactivity" above?

Reply:
OK, I've dropped the "two thirds" requirement here